niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/config/SignatureDependentPermissionConfiguration.java

```diff
@@ -93,17 +92,500 @@ public Optional<List<String>> getRuntimePermissions() {
 
     @Override
     public Optional<List<String>> getSignaturePermissions() {
+
         List<String> permissions = new ArrayList<>();
-        permissions.add(permission.RADIO_SCAN_WITHOUT_LOCATION);
```

Comment:
I don't think we want to remove these permissions unless the behavior has changed on Android T, and in that case, we still want to add these permissions for pre-T devices. This test is intended to run without the dependent runtime permissions granted to ensure these scan permissions are successful without the location permission; if the location permission is granted, then these permissions are never checked and the tester is not able to verify the permission behave as expected when only these signature permissions are granted.

Reply:
thanks. added

Reply:
Should the `NETWORK_SCAN` block below be restored as well to ensure this permission can be tested without a location permission?

Reply:
Thanks, fixed.

Reply:
Updates look good to me; thanks!